Helper/Data.php

```diff
@@ -162,4 +162,24 @@ public function apiKeyIsCorrect()
 
         return $keyIsCorrect;
     }
+
+    /**
+     * Get date in YYYY-MM-DD HH:MM:SS format
+     *
+     * @param string|null $date
+     *
+     * @return string|null
+     */
+    public function convertDeliveryDate(?string $date): ?string
```

Comment:
Waarom kan je hier null invullen? En waarom kan hij null terug geven?

Reply:
Deze hebben we besproken.
Het is nodig omdat in de magento admin mogelijk is om een zending aan te maken zonder dat daar een datum benoemd is.